src/core/dev_api/validation_util.hpp

```diff
@@ -56,12 +56,21 @@ OPENVINO_API std::shared_ptr<op::v0::Constant> get_constant_from_source(const Ou
 /// \brief Make scalar tensor which stores maximum value of ov::element::Type.
 /// \param et  Element type to get its maximum.
 /// \return Tensor with maximum value.
-Tensor make_tensor_of_max_value(const element::Type_t et);
+OPENVINO_API Tensor make_tensor_of_max_value(const element::Type_t et);
 
 /// \brief Make scalar tensor which stores minimum value of ov::element::Type.
 /// \param et  Element type to get its minimum.
 /// \return Tensor with minimum value.
-Tensor make_tensor_of_min_value(const element::Type_t et);
+OPENVINO_API Tensor make_tensor_of_min_value(const element::Type_t et);
+
+/// \brief Returns a Constant storing scalar value equal to std::numeric_limits<t>::max()
+OPENVINO_API std::shared_ptr<op::v0::Constant> get_constant_max_of_type(element::Type_t t);
+
+/// \brief Returns a Constant storing scalar value equal to std::numeric_limits<t>::min()
+OPENVINO_API std::shared_ptr<op::v0::Constant> get_constant_min_of_type(element::Type_t t);
+
+/// \brief Returns a Constant storing scalar value equal to std::numeric_limits<t>::lowest()
+OPENVINO_API std::shared_ptr<op::v0::Constant> get_constant_lowest_of_type(element::Type_t t);
```

Comment:
I think can be removed. Is only used ONNX FE, it handle creation of this kind constant locally

Reply:
Done